src/features/offer/components/SameArtistPlaylist/api/fetchOffersByArtist.ts

```diff
@@ -37,6 +41,10 @@ export const fetchOffersByArtist = async ({
       hitsPerPage: 30,
       attributesToRetrieve: [...offerAttributesToRetrieve, 'offer.artist', 'offer.ean'],
       attributesToHighlight: [], // We disable highlighting because we don't need it
+      aroundRadius: venueLocation ? convertKmToMeters(DEFAULT_RADIUS) : 'all',
+      aroundLatLng: venueLocation
+        ? `${venueLocation?.latitude}, ${venueLocation?.longitude}`
+        : undefined,
```

Comment:
Tu n'as plus besoin de vérifier `venueLocation` du coup.

```suggestion
      aroundLatLng: venueLocation
        ? `${venueLocation.latitude}, ${venueLocation.longitude}`
        : undefined,
```

voire 
```suggestion
      aroundLatLng: venueLocation
        ? [venueLocation.latitude, venueLocation.longitude].join(', ')
        : undefined
```

Reply:
je préfère la première option 👍